frontend/src/scenes/sessions/SessionsTable.tsx

```diff
@@ -187,7 +188,11 @@ export function SessionsTable({ personIds, isPersonPage = false }: SessionsTable
                 <Button onClick={nextDay} icon={<CaretRightOutlined />} />
             </Space>
 
-            {featureFlags['filter_by_session_props'] && (
+            {featureFlags['filter_by_session_props'] && user?.is_multi_tenancy && (
+                <SessionActionFilters actionFilter={actionFilter} updateActionFilter={updateActionFilter} />
+            )}
+
+            {featureFlags['filter_by_session_props'] && user?.is_multi_tenancy && (
```

Comment:
Nit: maybe put both filters under the same check? We're checking the exact same conditions

Reply:
Oops. Fixed.